prisma/schema.prisma

```diff
@@ -5,6 +5,7 @@ datasource db {
 
 generator client {
   provider = "prisma-client-js"
+  // previewFeatures = ["referentialIntegrity"]
```

Comment:
Oh wait we commented it now?

Reply:
oh wait my bad

Reply:
Resolved! Can you merge this please @Opeyem1a